src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const totalRent = 40 * days;
+
+  if (days >= 7) {
+    return totalRent - 50;
+  } else if (days >= 3) {
+    return totalRent - 20;
+  }
```

Comment:
[Checklist](https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md?rgh-link-date=2022-06-06T18%3A47%3A33Z) 1st item. You should create variables for main values - 20, 50, 40, 3, 7 they shouldn't be just magic numbers